src/main/kotlin/com/project/goms/domain/outing/usecase/ValidateOutingTimeUseCase.kt

```diff
@@ -0,0 +1,22 @@
+package com.project.goms.domain.outing.usecase
+
+import com.project.goms.global.annotation.UseCaseWithTransaction
+import java.time.DayOfWeek
+import java.time.LocalDate
+import java.time.LocalTime
+
+@UseCaseWithTransaction
+class ValidateOutingTimeUseCase {
+
+    fun execute() : Boolean{
+        val currentDate = LocalDate.now()
+        val currentTime = LocalTime.now()
+
+        if (currentDate.dayOfWeek != DayOfWeek.WEDNESDAY ||
+            (currentTime.isBefore(LocalTime.of(18, 50)) || currentTime.isAfter(LocalTime.of(19, 25)))) {
```

Comment:
LocalDate.now(), LocalTime.now()을 변수로 할당 하셔서 사용하셨으니, LocalTime.of()도 변수로 할당 하는게 더 코드가 깔끔해 보일거 같습니다.
```suggestion
        val currentDate = LocalDate.now()
        val currentTime = LocalTime.now()
        val startTime = LocalTime.of(18, 50)
        val endTime = LocalTime.of(19, 25)

        if (currentDate.dayOfWeek != DayOfWeek.WEDNESDAY || (currentTime.isBefore(startTime) || currentTime.isAfter(endTime))) {
```